1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
 ---
-post_title: Concepts
+post_title: DC/OS Architecture Concepts
 nav_title: Concepts
 menu_order: 5
 ---
-This page contains terms and definitions for DC/OS.
+## <a name="dcos-concepts"></a>DC/OS Concepts
 
-# <a name="adminrouter"></a>Admin Router
+DC/OS is an abstraction around pre-existing components. These terms are sometimes similar to component terms and sometimes they’re new terms meant to abstract organically grown component terms.
```

Comment:
Suggest `DC/OS is made up of many open source components, many of which existed before DC/OS. The terms used in this document may be similar to pre-existing terms that you are familiar with, however, they might be used in a different way with DC/OS.`